preupg/settings.py

```diff
@@ -43,6 +43,15 @@
 cache_dir = "/var/cache/preupgrade"
 log_dir = "/var/log/preupgrade"
 
+# file with meta info
+properties_ini = "properties.ini"
+# section inside ini file
+section_name = "preupgrade-assistant-modules"
+# key which specify source major OS version
+src_version_key ="srcMajorVersion"
+# key to specify destination major OS version
+dest_version_key ="destMajorVersion"
```

Comment:
Use dstMajorVersion istead of "dest". When you use already "src", "dst" is the best equivalent.